Controllers/HomeController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            ViewData["url"] = "https://bagcupid.com/";
+            ViewData["type"] = "website";
+            ViewData["title"] = "Bag Cupid";
+            ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+            return View();
+        }
+
+        [Route("app", Name = "app", Order = 0)]
+        public IActionResult app()
+        {
+            ViewData["url"] = "https://bagcupid.com/";
+            ViewData["type"] = "website";
+            ViewData["title"] = "Bag Cupid";
+            ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+            return View("Index");

```

Comment:
It can return index.html but i feel that is not necessary.
Razor view hasn't anything to conflict with, and your client developers can change directly inside .cshtml file. 

Look at View inside Views/Home/index.chtml it has all same content with only meta changes, which needs to be rendered dynamically.